src/Stratis.Bitcoin.IntegrationTests/Mempool/MemoryPoolTests.cs

```diff
@@ -184,31 +184,34 @@ public void TxMempoolBlockDoublespend()
                 }
 
                 // Test 1: block with both of those transactions should be rejected.
-                Block block = stratisNodeSync.GenerateBlockManually(spends);
+                var tipBeforeBlockCreation = stratisNodeSync.FullNode.Chain.Tip;
+                Assert.Throws<ConsensusException>(() => { Block block = stratisNodeSync.GenerateBlockManually(spends); });
                 TestHelper.WaitLoop(() => stratisNodeSync.FullNode.ConsensusManager().Tip.HashBlock == stratisNodeSync.FullNode.Chain.Tip.HashBlock);
```

Comment:
Why do we need to wait for syncing? Nothing has changed since the sync at line 164, no?

Reply:
@bokobza yeah those were there before? I can look at removing them. Let me check quick 👍

Reply:
@bokobza I checked and I could remove quiet a few one them 👍

Reply:
great